templates/sinatra_app/Gemfile.lock

```diff
@@ -24,7 +24,8 @@ GEM
     domain_name (0.5.20160310)
       unf (>= 0.0.5, < 1.0.0)
     eventmachine (1.2.0.1)
-    haml (4.0.7)
+    haml (5.0.0)
```

Comment:
I can not tell whether this only changes the gemfile.lock or if it also changes the Gemfile.